src/test/java/no/nav/pto/veilarbportefolje/postgres/PostgresServiceTest.java

```diff
@@ -53,36 +54,62 @@ public void setup() {
         oppfolginsbrukerRepositoryV2 = new OppfolginsbrukerRepositoryV2(db);
         arbeidslisteRepositoryV2 = new ArbeidslisteRepositoryV2(db);
 
-        VedtakstottePilotRequest vedtakstottePilotRequest =  mock(VedtakstottePilotRequest.class);
+        VedtakstottePilotRequest vedtakstottePilotRequest = mock(VedtakstottePilotRequest.class);
         veilarbVeilederClient = mock(VeilarbVeilederClient.class);
 
         postgresService = new PostgresService(vedtakstottePilotRequest, db, veilarbVeilederClient);
     }
 
     @Test
-    public void sok_resulterer_i_ingen_brukere(){
-        when(veilarbVeilederClient.hentVeilederePaaEnhet(any())).thenReturn(List.of("Z12345","Z12346"));
+    public void sok_resulterer_i_ingen_brukere() {
+        when(veilarbVeilederClient.hentVeilederePaaEnhet(any())).thenReturn(List.of("Z12345", "Z12346"));
 
         Filtervalg filtervalg = new Filtervalg().setFerdigfilterListe(List.of(UFORDELTE_BRUKERE));
-        postgresService.hentBrukere("1234",null, null,null, filtervalg, 0, 10);
+        postgresService.hentBrukere("1234", null, null, null, filtervalg, 0, 10);
 
     }
 
-
     @Test
-    public void sok_pa_tekst(){
-        when(veilarbVeilederClient.hentVeilederePaaEnhet(any())).thenReturn(List.of("Z12345","Z12346"));
+    public void sok_pa_tekst() {
+        when(veilarbVeilederClient.hentVeilederePaaEnhet(any())).thenReturn(List.of("Z12345", "Z12346"));
 
         Filtervalg teskt = new Filtervalg().setNavnEllerFnrQuery("test");
         Filtervalg fnr = new Filtervalg().setNavnEllerFnrQuery("123");
-        postgresService.hentBrukere("1234",null, null,null, teskt, 0, 10);
-        postgresService.hentBrukere("1234",null, null,null, fnr, 0, 10);
+        postgresService.hentBrukere("1234", null, null, null, teskt, 0, 10);
+        postgresService.hentBrukere("1234", null, null, null, fnr, 0, 10);
+    }
+
+    @Test
+    public void sok_pa_navn_case_insesitive() {
```

Comment:
@stianStensli can you please check tests, I am not sure what kind of scenarios to test beside those